internal/balancergroup/balancergroup.go

```diff
@@ -380,20 +380,26 @@ func (bg *BalancerGroup) Add(id string, builder balancer.Builder) {
 // UpdateBuilder updates the builder for a current child, starting the Graceful
 // Switch process for that child.
 //
-// TODO: update this API to take the name of the new builder instead.
-func (bg *BalancerGroup) UpdateBuilder(id string, builder balancer.Builder) {
+// Returns a non-nil error if the provided builder is not registered.
+func (bg *BalancerGroup) UpdateBuilder(id, builder string) error {
```

Comment:
You modified this to return an error, but don't check it at the call sites.  Even though those used to panic, the danger is that a "normal" error will be returned from here in the future, and so not checking it is problematic.

We should either panic if the builder isn't registered (as the previous behavior) or check the error at the call sites.

Reply:
Removed the `UpdateBuilder` call and replaced with a `ParseConfig` similar to the `gracefulswitch` balancer as discussed offline.